modules/doc/content/help/inl/hpc_remote.md

```diff
@@ -55,23 +55,16 @@ domain name in your command above because of the "Host" setting in your SSH conf
 
 ## SOCKS Proxy id=socks-proxy
 
-Adjust the SOCKS proxy settings for your +least+ favorite web browser to reflect the following
-settings:
+In order to access common HPC resources (hpcgitlab.hpc.inl.gov, hpcweb.inl.gov, etc) within a web browser, you must tunnel traffic to said resources using a SOCKS proxy. This is achieved by using a PAC (Proxy-Auto Configuration) file, which we have configured as:
 
-```bash
-localhost:5555
-```
+!listing moose/scripts/hpc_proxy.pac
 
-Choose your +least+ favorite, because once you adjust this setting, you will have to undo this
-setting once you disconnect for that browser to work again. There are other ways to switch back and
-forth by creating profiles. You may want to consult the Internet on how to set this up for your
-browser or OS. FireFox, for example, allows for custom profiles:
-[Firefox: Profile Manager](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles)
+Add the proxy configuration to your browser via the following url:
+
+```https://raw.githubusercontent.com/idaholab/moose/master/scripts/hpc_proxy.pac
+```
 
-Finally, you may want to enter a few hostnames that you do +not+ want to go resolve through your
-proxy.  These would typically be external INL systems that are resolvable on the Internet
-(e.g. +gitlab.software.inl.gov+, +www.inl.gov+). This will allow you to both internal and external
-INL sites when connected through the SOCKS proxy.
+Documentation is available for doing this within Firefox [here](https://support.mozilla.org/en-US/kb/connection-settings-firefox). Add the URL above within the "Automatic proxy configuration URL" box. We do not recommend utilizing Google Chrome with this functionality because it requires setting a system-wide proxy configuration.
```

Comment:
```suggestion
Documentation is available for using [Firefox](https://support.mozilla.org/en-US/kb/connection-settings-firefox). Add the URL above within the "Automatic proxy configuration URL" box. We do not recommend utilizing Google Chrome with this functionality because it requires setting a system-wide proxy configuration.
```